storm-core/src/clj/org/apache/storm/ui/helpers.clj

```diff
@@ -46,197 +46,3 @@
   (fn [req]
```

Comment:
Should we move this someplace else?  Or perhaps inline it into the servers that use it?